
AdaptiveOpenLocalPlannerROS:

  odom_topic: odom
  map_frame: odom
  obstacles_topic : obstacles

  extracted_path_rviz_topic : extracted_path_rviz
  current_pose_rviz_topic : current_pose_rviz
  roll_outs_rviz_topic : roll_outs_rviz
  weighted_trajectories_rviz_topic : weighted_trajectories_rviz
  safety_box_rviz_topic : safety_box_rviz
  car_footprint_rviz_topic : car_footprint_rviz
  box_obstacle_rviz_topic : box_obstacle_rviz
  cmd_vel_topic : cmd_vel

  planning_frequency : 1

  #  max speed that planner should not exceed
  max_speed : 2  
  # length of local trajectory roll outs
  max_local_plan_distance : 12
  #  distance between waypoints of local trajectory
  path_density : 0.3    
  # number of roll outs not including the center trajectory (this number should be even)   
  roll_outs_number : 8    
  #  length of car tip margin  
  sampling_tip_margin : 1.2  
  # length of roll in margin (??)  
  sampling_out_margin : 5   
  # distance between adjacent trajectories   
  roll_out_density : 0.3    
  roll_in_speed_factor : 0.2
  roll_in_margin : 6
  lane_change_speed_factor : 0.5
  horizon_distance : 15

  horizontal_safety_distance : 0.3
  vertical_safety_distance : 0.3
  max_steer_angle : 0.5
  min_speed : 0.2
  lateral_skip_distance : 10
# distance threshold for exiting following behavior
  min_following_distance : 5  
  #  distance threshold for entering following behavior
  max_following_distance : 10
  #  distance threshold for obstacle avoidance behavior
  min_distance_to_avoid : 8   

  vehicle_width : 1.25
  vehicle_length : 1.8
  wheelbase_length : 1.006
  turning_radius : 1.5
  safety_radius : 1.5

  smooth_data_weight : 0.45
  smooth_weight : 0.4
  smooth_tolerance : 0.1

  priority_weight : 0.9
  transition_weight : 0.9
  lat_weight : 1.5
  long_weight : 1
  collision_weight : 2
  curvature_weight : 1.2
  max_global_plan_lookahead_dist : 3.0
  global_plan_prune_distance : 1

  evaluate_path: false

  weighting: 0.1
  personal_learning_rate: 0.1
  global_learning_rate: 0.1
  cost_difference_boundary: 0.1
  max_interation: 10000
  max_linear_velocity: 1
  min_linear_velocity: 0
  max_angular_acceleration: 10
  min_angular_acceleration: -10
  max_linear_acceleration: 5
  min_linear_acceleration: -5

  path_divide_factor: 0.4